tests/functional/mesh/work_test.go

```diff
@@ -591,6 +591,62 @@ func TestRuntimeParams(t *testing.T) {
 	}
 }
 
+func TestKubeRuntimeParams(t *testing.T) {
```

Comment:
we should run the `checkSkipKube` in this test which skips a test if we set `SKIP_KUBE` in our environment

Reply:
thanks, fixed